utils/add_platform_rule.py

```diff
@@ -277,7 +277,7 @@ def clusterTestFunc(args):
         print('* Pushing image build to cluster')
         # execute the build_ds_container script
         buildp = subprocess.run(
-            ['utils/build_ds_container.sh', '-P', 'ocp4', '-P', 'rhcos4'])
+            ['utils/build_ds_container.py'])
```

Comment:
I think the python version still supports passing in different products. Should we maintain that?